Cargo.lock

```diff
@@ -2402,21 +2392,21 @@ dependencies = [
  "humantime-serde",
  "iroh",
  "iroh-metrics",
- "lru",
- "n0-future",
+ "lru 0.16.0",
```

Comment:
Another duplicate dependency.
Seems like that's due to the mainline/pkarr crates